src/client/firefox/commands.js

```diff
@@ -317,14 +320,48 @@ async function fetchSources() {
   return sources.map(source => createSource(source, { supportsWasm }));
 }
 
+/**
+ * Temporary helper to check if the current server will support a call to
+ * listWorkers. On Fennec 59 or older, the call will silently crash and prevent
+ * the client from resuming.
+ * XXX: Remove when FF59 for Android is no longer used or available.
+ *
+ * See https://bugzilla.mozilla.org/show_bug.cgi?id=1443550 for more details.
+ */
+async function checkServerSupportsListWorkers() {
+  const root = await tabTarget.root;
+  const deviceFront = await getDeviceFront(debuggerClient, root);
+  const description = await deviceFront.getDescription();
+
+  const isFennec = description.apptype === "mobile/android";
+  if (!isFennec) {
+    // Explicitly return true early to avoid calling Services.vs.compare.
+    // This would force us to extent the Services shim provided by
+    // devtools-modules, used when this code runs in a tab.
+    return true;
+  }
+
+  // We are only interested in Fennec release versions here.
+  // We assume that the server fix for Bug 1443550 will be uplifted to FF60.
+  // This means that we will not attempt to fetch workers for any early 60
+  // version such as 60.0b6.
+  const version = description.platformversion;
+  return Services.vc.compare(version, "60.0") >= 0;
```

Comment:
@jryans how do you feel about this comparison. Means pre-release v60 will not list workers either (which is not a big deal I think). But this way all 59.x versions should avoid the bug (assuming we can't uplift to 59).